src/Components/Web.JS/src/Platform/Mono/MonoPlatform.ts

```diff
@@ -200,12 +205,8 @@ function createEmscriptenModuleInstance(resourceLoader: WebAssemblyResourceLoade
     // Fetch the assemblies and PDBs in the background, telling Mono to wait until they are loaded
     // Mono requires the assembly filenames to have a '.dll' extension, so supply such names regardless
     // of the extensions in the URLs. This allows loading assemblies with arbitrary filenames.
-    resourceLoader.loadResources(resources.assembly, filename => `_framework/_bin/${filename}`)
-      .forEach(r => addResourceAsAssembly(r, changeExtension(r.name, '.dll')));
-    if (resources.pdb) {
```

Comment:
Do we still need this logic?

Reply:
The `if` went away - ah nvm, I see it was replaced with `resources.pdb || { }` 👍